charts/linkerd-control-plane/values.yaml

```diff
@@ -321,23 +337,23 @@ proxyInjector:
   # -- Namespace selector used by admission webhook.
   namespaceSelector:
     matchExpressions:
-    - key: config.linkerd.io/admission-webhooks
-      operator: NotIn
-      values:
-      - disabled
-    - key: kubernetes.io/metadata.name
-      operator: NotIn
-      values:
-      - kube-system
-      - cert-manager
+      - key: config.linkerd.io/admission-webhooks
+        operator: NotIn
+        values:
+          - disabled
+      - key: kubernetes.io/metadata.name
+        operator: NotIn
+        values:
+          - kube-system
+          - cert-manager
```

Comment:
While I prefer this indentation style, personally, the rest of the helm manifests styles don't have this extra indentation. Remaining consistent (i.e. reverting this change) is probably best for now. Here and below...

Reply:
VS Code made these changes, is there a setting I should be using to keep it from rewriting the indentation?

Reply:
For now you may want to set
```
"yaml.format.enable": false
```

in your settings.json. But sometime soon we should probably update these manifests to be consistent with the formatting tools

Reply:
https://github.com/redhat-developer/vscode-yaml/discussions/839